### ISSUE TITLE
Some previously translated strings are no longer translated

### ISSUE THREAD

**Issue by jbphet** [phetsims/fraction-matcher#107]
The level selection screen for the published Farsi (Persian) version of Fraction Matcher looks like this:

![image](https://user-images.githubusercontent.com/3082831/53672457-d96e4700-3c3f-11e9-920f-43c197201ac3.png)

On the current master version, using locale=fa, it looks like this:

![image](https://user-images.githubusercontent.com/3082831/53672500-0fabc680-3c40-11e9-8c62-54ba4f68c411.png)

The reason that English words are now appearing is that a number of strings were moved from the fraction-matcher repo to fractions-common during the recent work on the fractions suite, and the translated strings weren't moved over.  This should probably be fixed, otherwise the next time Fraction Master is published off of master, it may cause existing translations to fall back to English in several places as seen above.

I'm guessing it would be a couple hours of work max to either propagate the strings manually or write a script to do it.  Either @jonathanolson or I could do it, or perhaps someone who we want to get a better understanding of how the translation utility works.  Assigning to @ariel-phet for prioritization and assignment.

**Comment by jbphet** [comment]
I was just talking with @KatieWoe, and she said she was testing a dev version of this sim, so I'm going to add the "blocks-publication" label so that we make sure that this gets some attention before the sim is republished off of master.

**Comment by ariel-phet** [comment]
@jbphet please go ahead and do this work.

**Comment by jbphet** [comment]
I wrote a very special-purpose script that copied over the strings that were in fraction-matcher that weren't yet in fractions-common.  Once done, I tested this by running the RequireJS version of fraction-matcher with `locale=fa` (Persian), and found that the strings depicted in the screenshot above, as well as the "ok", "timeNumberSec", "myMatches", and "ok" strings all appeared correctly in the translated version.

No redeployment should be required, since this wasn't going to show up until fraction-master was redeployed from master, so I think that's all that needs to be done.

Assigning to @jonathanolson to review my work on this and to close or assign back to me as appropriate.
